homeassistant/components/philips_js/config_flow.py

```diff
@@ -53,34 +67,99 @@ async def async_step_import(self, conf: Dict[str, Any]):
             }
         )
 
+    async def _async_create_current(self):
+
+        system = self._current[CONF_SYSTEM]
+        return self.async_create_entry(
+            title=f"{system['name']} ({system['serialnumber']})",
+            data=self._current,
+        )
+
+    async def async_step_pair(self, user_input: Optional[Dict] = None):
+        """Attempt to pair with device."""
+        assert self._hub
+
+        errors = {}
+        schema = vol.Schema(
+            {
+                vol.Required(CONF_PIN): str,
+            }
+        )
+
+        if not user_input:
+            try:
+                self._pair_state = await self._hub.pairRequest(
+                    CONST_APP_ID,
+                    CONST_APP_NAME,
+                    platform.node(),
+                    platform.system(),
+                    "native",
+                )
+            except PairingFailure as exc:
+                LOGGER.debug(str(exc))
```

Comment:
The exception instance has a magic method `__str__`. We don't need to copy to string.